test/e2e/performanceprofile/functests/4_latency/latency.go

```diff
@@ -455,7 +456,9 @@ func createLatencyTestPod(testPod *corev1.Pod, node *corev1.Node, logName string
 		testlog.Error(err)
 		logEventsForPod(testPod)
 	}
-	Expect(err).ToNot(HaveOccurred())
+	updatedPod := &corev1.Pod{}
+	testclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(testPod), updatedPod)
+	Expect(err).ToNot(HaveOccurred(), "Expected the pod to reach running phase but its current phase is %s", updatedPod.Status.Phase)
```

Comment:
this test looks wrong, don't we need to check the `updatedPod` phase?

Reply:
the status is checked here:
https://github.com/openshift/cluster-node-tuning-operator/pull/368/files#diff-41952f42e8241558ea4f4d9cea270beab57c936f488bc6ff41984c20546278a6L453
maybe the name of the variable is misleading here, as it is not updated pod in terms of specs but of status

Reply:
I see, ok I'll make that a separate klog error

Reply:
ok, my too terse statement may have been misleading, sorry about that. The problem here is intermixing the check to `err` and the message regarding the updated pod status. it should be probably be better fixed in `pods.WaitForPhase`.